### PR TITLE
Flight search page

### PR DESCRIPTION
**Pull request by vi-nguyen-12**
I am still working on this. Please have a look and comment any ideas, thank you.

### CODE REVIEW

File: client/src/components/Search/index.js
Comment (sina-jamshidi) [inline client/src/components/Search/index.js]: I wonder if your sizing problem is because your dividers are not inside any Grid `item`s, and so the Grids all add up to 12, but there's the extra dividers and they throw it off. Try putting the dividers inside the Grid `item`s and see if that works. For example:
```
                    <Grid item xs={12} sm={2}>
                       <Divider orientation="vertical" flexItem />
                       <Button variant='contained'> Search</Button>
                    </Grid>   
```

File: client/src/components/Search/styled.js
Comment (sina-jamshidi) [inline client/src/components/Search/styled.js]: for the sake of consistency across the app, let's use `makeStyles` to handle the styling

File: server/.env
Comment (sina-jamshidi) [inline server/.env]: can remove this

File: client/src/components/Search/index.js
Comment (vi-nguyen-12) [reply client/src/components/Search/index.js]: It works for the layout now, but in the meantime, the line of dividing disappears.

File: server/.env
Comment (vi-nguyen-12) [reply server/.env]: Okay, I will

File: client/src/components/Search/styled.js
Comment (vi-nguyen-12) [reply client/src/components/Search/styled.js]: Got it, I will convert them to makeStyles.

File: server/.env
Comment (sina-jamshidi) [inline server/.env]: This should be git-ignored

**Review by sina-jamshidi — APPROVED** [review]
This seems okay so far but let's wait until after the standup to merge

**Pull request merged**
